mmda/types/box.py

```diff
@@ -18,11 +18,23 @@ class Box:
     h: float
     page: int
 
-    # TODO[kylel]: make this more minimal
-    def to_json(self) -> Dict:
-        return dict(l=self.l, t=self.t, w=self.w, h=self.h, page=self.page)
+    def to_json(self) -> List:
+        return [self.l, self.t, self.w, self.h, self.page]
+        # return dict(l=self.l, t=self.t, w=self.w, h=self.h, page=self.page)
 
     @classmethod
-    def from_json(cls, box_dict) -> "Box":
-        return Box(l=box_dict['l'], t=box_dict['t'], w=box_dict['w'], h=box_dict['h'], page=box_dict['page'])
+    def from_json(cls, box_coords: List) -> "Box":
+        l, t, w, h, page = box_coords
+        return Box(l=l, t=t, w=w, h=h, page=page)
+        # return Box(l=box_coords['l'], t=box_coords['t'], w=box_coords['w'], h=box_coords['h'], page=box_coords['page'])
 
+    @classmethod
+    def small_boxes_to_big_box(cls, boxes: List['Box']) -> 'Box':
```

Comment:
I think we need to have a specific documentation page that specifies several different ways for "combining" boxes. 
I would call it "union" somehow..

Reply:
i previously had it called `union` but it wasn't descriptive enough

but yes, we'll have different combining strategies